### ISSUE TITLE
Env variable instead of `submission=True` in python

### ISSUE THREAD

**Issue by artemisart** [cs4246/docker-base#1]
The docker environment could set an env var, and then the python script files could check e.g. `os.environ.get('CS4246_SUBMISSION', False)`, that way there wouldn't be any change to make to the script before uploading and no possibility of error.
Btw: could the homeworks be uploaded to github? I would like to make some PR :)

**Comment by artemisart** [comment]
An alternative would be testing for the import:
```python
try:
	from runner...
	SUBMISSION = True
except ImportError:
	SUBMISSION = False
```
But the env var solution would be preferable I believe.

**Comment by rizkiarm** [comment]
Yes, that's true. In fact, the agent template for HW3 and mini-project don't have the flag (I was the one setting them up). I have advised the other TAs that setup HW1 and HW2 to remove them from the beginning, but apparently, it persists at the end.

What do you mean by uploading homework to GitHub?

**Comment by artemisart** [comment]
I mean the templates files (HW1.zip and HW2.zip we got) so that we could make PR and issues on them if needed.
